content/spaces/observability.md

```diff
@@ -49,18 +49,25 @@ Prometheus may not meet an Operators business or technology requirements.
 Upbound advises operators to consider what metrics and observability stack meet
 meet their needs before running in production.
 
-The recommended way to install Prometheus is through [the official helm charts](https://github.com/prometheus-community/helm-charts)
+The recommended way to install Prometheus is through
+[the official helm charts](https://github.com/prometheus-community/helm-charts).
 
+Add the Prometheus chart repository:
 ```shell
-# Add the prometheus helm char
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-# Install prometheus, kube-state-metrics and prometheus. See
-# the official docs the "offical helm charts" link above.
+```
+
+Install Prometheus, Alertmanager, Prometheus node exporter, kube-state-metrics,
```

Comment:
I'm not sure how to tell the validation check that it is in fact spelled Alertmanager 🤔